Miredot license Issue

![WebAPI Miredot License](https://user-images.githubusercontent.com/37068413/79191532-c5949d00-7e61-11ea-84c6-9e2324ff4368.PNG)

### **> [ERROR] License key expired on 31 3 2020**

When building the WebAPI .war file using Maven, an error has occurred

--- a/pom.xml
+++ b/pom.xml
@@ -381,7 +381,7 @@
           </execution>
         </executions>
         <configuration>
-          <licence>cHJvamVjdHxvcmcub2hkc2kuV2ViQVBJfDIwMjAtMDMtMzF8dHJ1ZXwtMSNNQzBDRkNNQ2R2ZEhjbDg0Z2tONVI3bHlmQ2Y4SnNkYUFoVUFoMkdtOUVFdy9RelpLVzM1V2p6a2FsdzJZV009</licence>
+          <licence>cHJvamVjdHxvcmcub2hkc2kuV2ViQVBJfDIwMjUtMDEtMDF8ZmFsc2V8LTEjTUN3Q0ZDZkowMkZrVmVSeVlBazZVbTBmNThIbE15SjRBaFEwcVZyRUZuamZMTzJ0KzFtR3E3U3lMMkFESHc9PQ==</licence>
           <restModel>
             <httpStatusCodes>
               <httpStatusCode>

